build-template/build.cmd

```diff
@@ -16,11 +16,19 @@ copy %CACHED_NUGET% .nuget\nuget.exe > nul
 
 :restore
 IF EXIST packages\KoreBuild goto run
-.nuget\NuGet.exe install KoreBuild -ExcludeVersion -o packages -nocache -pre
+IF DEFINED BUILDCMD_RELEASE (
+	.nuget\NuGet.exe install KoreBuild -version 0.2.1-%BUILDCMD_RELEASE% -ExcludeVersion -o packages -nocache -pre
```

Comment:
Trying to think through what the versioning scheme for KoreBuild would be once we hit RTM. `0.2.1-RTM10` works, but I'm not sure how we'd express the betas for `> 1.0` (`0.2.1-BETA11` would be problematic since it's smaller than `0.2.1-beta9`). Perhaps we'll have to update build.cmd in all our repos once we hit RTM to point to `0.2.2-%BUILDCMD_RELEASE%`?